### PR TITLE
docs: governor update

### PR DESCRIPTION
**Pull request by wonge97**
reflect new governor auto-release enqueued messages feature

### CODE REVIEW

**Review by jumpsiegel — APPROVED** [review]

File: whitepapers/0007_governor.md [hunk 1]
```diff
@@ -33,13 +43,25 @@ The checks performed include:
...
 3. Is the message a known type that transfers value?
 4. Is the token transferred listed within `mainnet_tokens.go`?
-5. Will the transfer amount bring the total notional value transferred within a specified time frame over the limit?
+5. Is the transaction a “large” transaction (ie. greater than or equal to `bigTransactionSize` for this chain)?
+6. Is the transaction a “small” transaction (ie. less than `bigTransactionSize` for this chain)?
+
+The above checks will produce 3 possible scenarios:
+
+- **Non-Governed Message**: If a message does not pass checks (1-4), `ChainGovernor` will indicate that the message can be published.
+- **Governed Message (Large)**: If a message is “large”, `ChainGovernor` will wait for 72hrs before signing the VAA and place the message in a queue.
```
Comment (SEJeff) [inline whitepapers/0007_governor.md:52]: Change 72 --> 24 per #1577

File: whitepapers/0007_governor.md [hunk 2]
```diff
@@ -16,9 +16,19 @@ Bridge security is incredibly high stakes — beyond core trust assumptions and
...
+Each individual Guardian within the Guardian network should employ a set of strategies to verify the validity of a VAA. The Governor is designed to check VAAs that transfer tokens by enforcing limits on the notional value that can be transferred from a given chain over a specific period of time. 
 
-There are many other potential variations on the notional value limit and time frame considered (i.e. 4 hour window, 12 hour window, max single transaction size) — this initial implementation is for a 24-hour window with a custom limit per chain that is informed by data-driven analysis from recent chain activity.
+The current implementation works on two classes of transaction (large and small) and current configuration can be found [here](https://github.com/wormhole-foundation/wormhole/blob/dev.v2/node/pkg/governor/mainnet_chains.go):
+
+- **Large Transactions**
+    - A transaction is large if it is greater than or equal to the `bigTransactionSize` for a given origin chain.
+    - All large transactions will have a mandatory 72-hour finality delay and will have no affect on the `dailyLimit`.
```
Comment (SEJeff) [inline whitepapers/0007_governor.md:25]:
```suggestion
    - All large transactions will have a mandatory 24-hour finality delay and will have no affect on the `dailyLimit`.
```

See #1577

File: whitepapers/0007_governor.md (hunk 1 above)
Comment (evan-gray) [reply whitepapers/0007_governor.md]: fixed

File: whitepapers/0007_governor.md (hunk 2 above)
Comment (evan-gray) [reply whitepapers/0007_governor.md]: fixed

**Review by evan-gray — APPROVED** [review]

**Review by ckeun — APPROVED** [review]

**Pull request merged**